mobify/astro-scaffold
# Astro on npm

Reviewers: Anyone on Astro!
JIRA: https://mobify.atlassian.net/browse/HYB-646
Linked PRs: https://github.com/mobify/astro/pull/399

## Changes
- Brings the Astro SDK dependancy through npm rather than through git submodules

## How to test-drive this PR
- Checkout this branch, run `npm install`, make sure everything still works!
- Follow instructions in README `Developing against `develop` of astro` and ensure you can successfully develop against a local version of Astro.

## TODOS:
- [x] Upload the Astro folder to npm
- [x] Prompt the generator to show the license
- [x] Update the tutorial to use npm
- [x] Change works in both Android and iOS (SJ: I haven't fixed Android yet!)
- [x] Update the generator
- [x] Update the astro-sdk dependency to 0.7.0 (not -dev)
